packages/nimble/inst/include/nimble/NimArr.h

```diff
@@ -86,10 +86,55 @@ class NimArr<1, T> : public NimArrBase<T> {
     return(*this);
   }
 
-  NimArr<1, T> &operator=(const NimArr<1, double> &other) {return(templateCopyOperator<double>(other));}
-  NimArr<1, T> &operator=(const NimArr<1, int> &other) {return(templateCopyOperator<int>(other));}
-  NimArr<1, T> &operator=(const NimArr<1, bool> &other) {return(templateCopyOperator<bool>(other));}  
+  /* NimArr<1, T> &operator=(const NimArr<1, double> &other) {return(templateCopyOperator<double>(other));} */
```

Comment:
Let me know when you merge this PR. It will conflict with my style-cleanup PR #423, so I'll hold off merging until you've merged this (to save you the conflict resolution effort).